sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/GraphClient.java

```diff
@@ -54,17 +62,20 @@ public GraphClient(AADAuthenticationProperties properties) {
             memberships.getValue()
                        .stream()
                        .filter(this::isGroupObject)
-                       .map(membership -> {
-                           if (properties.getUserGroup().getEnableFullList()) {
-                               return Arrays.asList(membership.getObjectID());
-                           } else {
-                               return Arrays.asList(membership.getDisplayName(), membership.getObjectID());
-                           }
-                       }).forEach(groups::addAll);
+                       .map(membership -> Arrays.asList(membership.getObjectID()))
+                       .forEach(groupsId::addAll);
+            memberships.getValue()
+                       .stream()
+                       .filter(this::isGroupObject)
```

Comment:
Can we use the following code to avoid iterating twice?
```
for () {
    if (isGroupObject()) {
        // ...
    } else {
        // ...
    }
}
```